ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -359,7 +360,7 @@ object BigDiffy extends Command with Serializable {
     keyFn: TableRow => MultiKey,
     diffy: TableRowDiffy,
     ignoreNan: Boolean = false
-  ): BigDiffy[TableRow] = {
+  )(implicit coder: Coder[TableRow]): BigDiffy[TableRow] = {
```

Comment:
if you want to avoid changing this API signature, you could remove this implicit coder from the signature and just add `implicit val trCoder = Coder.beam(TableRowJsonCoder.of())` to the method body! Or alternately, just do `diff(...)(ClassTag[TableRow], Coder.beam(TableRowJsonCoder.of())` 👍